Update Environment Canada integration alerts

SapphirusBeryl opened the pull request:
Fixes https://github.com/MrBartusek/MeteoalarmCard/issues/301

SapphirusBeryl committed:
Update Environment Canada integration alerts

SapphirusBeryl committed:
Support uncoded alerts

SapphirusBeryl committed:
Update Environment Canada level type critera

nickvin commented:
Any reason why this isn't getting merged?
The card hasn't been functional for Environment Canada users for a while now.
Would be great to get this fix...

SapphirusBeryl committed:
Fix severe alert criteria

Environment Canada seems to no longer use adjectives describing severity
in their alert names.

SapphirusBeryl commented:
> Any reason why this isn't getting merged? The card hasn't been functional for Environment Canada users for a while now. Would be great to get this fix...

I'd respectfully ask to have some patience and not bug the maintainer. 


<details>
  <summary><b>Note</b>: It's possible to build and install MeteoalarmCard manually with these patches if you are unable to wait:</summary>
  
## Instructions for the uninitiated

Expectation is you are at least able to make basic use of a Linux CLI. Beyond this, I cannot provide any assistance, as it falls outside the scope of this PR. 

The instructions are as follows:

1. Run `$ git clone https://github.com/SapphirusBeryl/MeteoalarmCard/ && cd MeteoalarmCard && git switch update/environment-canada-integration`
2. Then `$ npm install && npm run build`
3. Move `dist/meteoalarm-card.js` to `www/custom/` in your HomeAssistant configuration directory
4. Navigate on your HomeAssistant instance to `https://home-assistant/config/lovelace/resources`
5. Then add a custom resource with the URL `/local/custom/meteoalarm-card.js` and ensure the Resource type is set to JavaScript module
6. Clear your browser's cache and reload the frontend
</details>

SapphirusBeryl committed:
Fix thunderstorm alert criterion
